application/config.json.template

```diff
@@ -88,4 +88,9 @@
     ],
     "logInfoChannelWebhook": "<put_your_webhook_here>",
     "logErrorChannelWebhook": "<put_your_webhook_here>"
+    "oofsAndLmaos": {
+        "oofEmojiName": ":oof:"
+        "lmaoEmojiName": ":lmao"
```

Comment:
use a list instead, and its without `:` (see `upVoteEmoteName` further up)

Reply:
I'll remove the `:`
However, even upVoteEmoteName usesobjects and not lists.
```json
"suggestions": {
        "channelPattern": "tj_suggestions",
        "upVoteEmoteName": "peepo_yes",
        "downVoteEmoteName": "peepo_no"
    }
    ```
    Aren't lists wrapped with `[]`?

Reply:
yes. in-code u will use a `List<String>` and in the json its written `"emotes": ["foo", "bar", "baz"]`

Reply:
Yes, so how would a list be useful here?
Suggestions config itself uses objects, like I mentioned earlier.

Reply:
because it only is meant to support two emotes. but this one here is supposed to be configurable without code-changes. what if i want to change it to one emote? or to 10 emotes? it should support an arbitrary list of emotes that trigger the feature

Reply:
Oh, so I will only have a single list of emoji names instead of two emote names?
That would reduce the number of entries in the config to 2 from 3, which may not warrant having a wrapper class of `StarboardConfig`

Reply:
im okay with keeping the config class

Reply:
btw, please also ensure the code works if the list is empty. im planning to be able to deactivate the feature by having an empty list